rebar.config

```diff
@@ -23,9 +23,12 @@
 {profiles, [
   {test, [
     {deps, [
-      {katana_test, "0.1.1"},
-      {mixer, "0.1.5", {pkg, inaka_mixer}},
-      {meck, "0.8.7"}
+      {katana_test, {git, "https://github.com/inaka/katana-test", {branch, "master"}}},
```

Comment:
We don't use `master` as a branch for depenencies, even for test ones.
You either need to use a release that's on hex (the recommended one would be `1.0.0`) or a commit SHA.